nifi-commons/nifi-record/src/test/java/org/apache/nifi/serialization/record/ResultSetRecordSetTest.java

```diff
@@ -241,6 +243,45 @@ public void testCreateRecord() throws SQLException {
         assertEquals(bigDecimal5Value, record.getValue(COLUMN_NAME_BIG_DECIMAL_5));
     }
 
+    @Test(expected = SQLException.class)
```

Comment:
To provide more precision in testing, the `assertThrows()` method supports indicating the expected exception on the particular method being called.  In addition, should the expected exception be `SQLFeatureNotSupportedException`?